ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -71,13 +77,20 @@ export class Burn extends IronfishCommand {
       account = defaultAccount.name
     }
 
+    const confirmations = flags.confirmations
+    if (confirmations !== undefined && confirmations < 0) {
+      this.log('Number of block confirmations must be non-negative')
+      this.exit(1)
+    }
```

Comment:
you don't exactly need to add this, since negative confirmations can kind of just work too.

Reply:
removed the confirmation validations